vite.config.ts

```diff
@@ -41,7 +41,10 @@ export default defineConfig(({ mode }) => {
         '@': resolve(__dirname, './resources/js'),
       },
     },
-    test: {},
+    test: {
+      environment: 'jsdom',
```

Comment:
Unexpectedly, this seems to work without an explicit `devDependency` for `jsdom`.

`npm ls jsdom` shows we are pulling in `jsdom` transitively through `vitest`.